msu/utils/text.nut

```diff
@@ -28,4 +28,92 @@
 	{
 		return this.color(::Const.UI.Color.NegativeValue, _string);
 	}
+
+	function colorizeValue( _value, _kwargs = null )
+	{
+		local kwargs = {
+			AddSign = false,
+			CompareTo = 0,
+			Invert = false,
```

Comment:
Should this be `InvertColor` or just `Invert`?

Reply:
`InvertColor` imo